docs/appendices/release-notes/unreleased.rst

```diff
@@ -111,4 +115,4 @@ Fixes
 
 - Fixed an issue causing nested join statements using the ``NESTED LOOP`` plan
   to return incorrect results in some scenarios when issued on a multi-node
-  cluster.
\ No newline at end of file
+  cluster.
```

Comment:
unrelated?

Reply:
most likely ... it is just a removed whitespace at the end, which afaik is correct anyway.